static/app/views/performance/transactionSummary/transactionOverview/trendChart/index.tsx

```diff
@@ -9,24 +9,40 @@ import {getInterval, getSeriesSelection} from 'sentry/components/charts/utils';
 import {normalizeDateTimeParams} from 'sentry/components/organizations/pageFilters/parse';
 import QuestionTooltip from 'sentry/components/questionTooltip';
 import {t} from 'sentry/locale';
-import {OrganizationSummary} from 'sentry/types';
+import {Organization, OrganizationSummary, Project} from 'sentry/types';
 import {getUtcToLocalDateObject} from 'sentry/utils/dates';
+import EventView from 'sentry/utils/discover/eventView';
+import TrendsDiscoverQuery from 'sentry/utils/performance/trends/trendsDiscoverQuery';
+import {decodeScalar} from 'sentry/utils/queryString';
 import useApi from 'sentry/utils/useApi';
 import {useLocation} from 'sentry/utils/useLocation';
 import useRouter from 'sentry/utils/useRouter';
 
-import {TrendFunctionField} from '../../../trends/types';
-import {generateTrendFunctionAsString} from '../../../trends/utils';
+import {
+  NormalizedTrendsTransaction,
+  TrendChangeType,
+  TrendFunctionField,
+  TrendView,
+} from '../../../trends/types';
+import {
+  generateTrendFunctionAsString,
+  getSelectedQueryKey,
+  modifyTrendView,
+  normalizeTrends,
+} from '../../../trends/utils';
```

Comment:
absolute path here would make import path a bit more readable. we try to use absolute imports for files that are in a different folder